dask/tests/test_utils.py

```diff
@@ -67,6 +68,30 @@ def test_ensure_bytes_pyarrow_buffer():
     assert isinstance(result, bytes)
 
 
+def test_ensure_unicode():
+    data = [b"1", "1", memoryview(b"1"), bytearray(b"1"), array.array("b", [49])]
```

Comment:
I'm wondering why the string representation of `array.array("b", [49])` should be `"1"`. Looking at the other objects in `data`, I see that `"1"` is more-or-less the underlying information contained in the object, but I'm surprised to see the same for `array.array("b", [49])`. This is mostly just for my own understanding

Reply:
Maybe this helps clarify it? This is coming from [ASCII]( https://en.wikipedia.org/wiki/ASCII#Printable_characters ) originally.

```python
In [1]: ord("1")
Out[1]: 49
```

Reply:
Looks like `array("B", b"1")` already works. So switched to that